urdf/package.xml

```diff
@@ -28,15 +28,16 @@
   <build_depend>urdf_parser_plugin</build_depend>
   <build_depend>pluginlib</build_depend>
   <build_depend>cmake_modules</build_depend>
-  <build_depend>rostest</build_depend>
   <build_depend>tinyxml</build_depend>
 
-  <run_depend>liburdfdom-dev</run_depend>
-  <run_depend>liburdfdom-headers-dev</run_depend>
-  <run_depend>rosconsole_bridge</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>urdf_parser_plugin</run_depend>
-  <run_depend>pluginlib</run_depend>
-  <run_depend>tinyxml</run_depend>
+  <exec_depend>liburdfdom-dev</exec_depend>
+  <exec_depend>liburdfdom-headers-dev</exec_depend>
+  <exec_depend>rosconsole_bridge</exec_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>urdf_parser_plugin</exec_depend>
```

Comment:
Looks like this package only has a single header in it, so I don't think it needs to be an `exec_depend`.